Getting Started with Form (#1485)
DevExpress/devextreme-documentation: concepts/05 UI Components/Form/01 Getting Started with Form/40 Organize the Form Items/10 In Columns.md

> The Form can organize items in a fixed number of columns or automatically adjust it to the screen width. To set the fixed number of columns, assign it to the [colCount](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#colCount) property as done in the code below. If the number should be auto-adjusted, configure the [screenByWidth](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#screenByWidth) and [colCountByScreen](/Documentation/ApiReference/UI_Components/dxForm/Configuration/colCountByScreen/) properties. 

Reviewer: What does 'it' mean? "Organize items ... or adjust it "... Maybe "adjust the layout"? 

 ```suggestion
The Form can organize items into a fixed number of columns or automatically adjust the layout based on the screen width. To set a fixed number of columns, initialize the [colCount](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#colCount) property as done in the code below. To create an adaptive layout, configure the [screenByWidth](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#screenByWidth) and [colCountByScreen](/Documentation/ApiReference/UI_Components/dxForm/Configuration/colCountByScreen/) properties. 
```

> The Form can organize items into a fixed number of columns or automatically adjust the layout based on the screen width. To set a fixed number of columns, initialize the [colCount](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#colCount) property as done in the code below. To create an adaptive layout, configure the [screenByWidth](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#screenByWidth) and [colCountByScreen](/Documentation/ApiReference/UI_Components/dxForm/Configuration/colCountByScreen/) properties. 

Reviewer 2: ```suggestion
The Form can organize items into a fixed number of columns or automatically adjust the layout based on the screen width. To specify the number of columns, initialize the [colCount](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#colCount) property as shown in the code below. To create an adaptive layout, configure the [screenByWidth](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#screenByWidth) and [colCountByScreen](/Documentation/ApiReference/UI_Components/dxForm/Configuration/colCountByScreen/) properties. 
```

Author: We have two types of the Form layout: one has the fixed number of columns regardless of the screen width, another one has different number of columns for different devices. How about the following:
'To keep the fixed number of columns, inintialize...'

> An item can span multiple columns. The example below sets [colSpan](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/#colSpan) property for the `Notes` item so that it spans two columns.

Reviewer 2: ```suggestion
An item can span multiple columns. The example below sets the [colSpan](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/#colSpan) property for the `Notes` item so that it spans two columns.
```

Reviewer 2: "sets the [colSpan] property" Sets it to what?

Author: will fix it, 

Sets the colSpan property for the `Notes` items to `2` so that it spans two columns.

> An item can occupy more than one column. For this, specify the [colSpan](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/#colSpan) property. In the example below, the `Notes` item spans two columns:

Reviewer: "For this" sounds like a literal "russian-to-english" translation. Seems weird to me. 

```suggestion
An item can span multiple columns. The example below sets [colSpan](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/#colSpan) property for the `Notes` item so that it spans two columns.
```
